package.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Your env problem might be due to you missing a setting in your package.json, please reference this package.json to see the difference, namely, the property that says "jest".
https://github.com/codefellows/seattle-javascript-401d23/blob/master/back-end/14-relationship-modeling/kanban-api/package.json